tools/cli/commands/generate.js

```diff
@@ -402,12 +402,18 @@ async function mirrorRepo( composerJson, name, org = 'Automattic' ) {
 			message: 'What name do you want to use for the repo?',
 			when: newAnswers => exists && ! newAnswers.useExisting, // When there is an existing repo, but we don't want to use it.
 		},
+		{
+			type: 'confirm',
+			name: 'autotagger',
+			default: true,
+			message: 'Enable autotagger for the mirror repo?',
```

Comment:
Should we add any text for what autotagger is?

Reply:
Yeah, I'd love to - same for the other choices. I wasn't sure how to put it succinctly. [The monorepo doc](https://github.com/Automattic/jetpack/blob/master/docs/monorepo.md#autotagger) says:

```
If .extra.autotagger is set to a truthy value in the project's composer.json, a GitHub Action will be included in the mirror repo that will read the most recent version from the mirrored CHANGELOG.md in each push to master, and create the tag if that version has no prerelease or build suffix.
```

Maybe:

message: 'Have mirror repo automatically create new tags based on version specified in CHANGELOG.md?'